src/index.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import App from "./App";
+// import App from "./App";
 import "./styles.css";
 import * as serviceWorker from "./serviceWorker";
+import HomePage from "./HomePage";
 
-ReactDOM.render(<App />, document.getElementById("root"));
```

Comment:
Let the base component be `<App/>`. What i want you to do is add a Router inside of `<App/>` to the project, making `localhost:300
0/home` load the <Homepage /> and `localhost:3000/dashboard` load the dashboad.